CHANGELOG.md

```diff
@@ -1,129 +1,112 @@
-Changes by Version
-==================
+# Changes by Version
```

Comment:
can we skip .md files for now? This file post pretty looks incorrectly formatted.

Reply:
Yes. Also, @black-adder, if you have any formatting preferences, please let me know. I've just used Prettier's default ones, as I don't have strong preferences for formatting, other than it be done automatically.